UserDetails/views_user_settings.py

```diff
@@ -0,0 +1,41 @@
+from django.contrib import messages
+from django.contrib.auth.mixins import LoginRequiredMixin
+from django.shortcuts import render, redirect
+from django.views.generic import TemplateView
+from django.utils.translation import gettext as _
+
+from .forms import UserForm, DiningProfileForm, AssociationLinkForm
+
+
+class Settings_Profile_View(LoginRequiredMixin, TemplateView):
```

Comment:
Please use the Python naming conventions.

Reply:
Oops, sorry. Renamed